dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -212,10 +212,18 @@ TUMultiplicity :
 
 // Commons
 
+TUModifiers:
```

Comment:
We should consider that using a common enum rule for all possible modifiers would result in more intuitive error messages. For example with this solution, in case of a duplicate modifier we can produce only a syntax error instead of an error message mentioning the duplication, where the latter would be achievable with enum rules and validation.